### ISSUE TITLE
Sonarqube cleanup

### ISSUE THREAD

**Issue by spbolton** [dotCMS/core#29261]
Sonarqube reliability rating causing issues blocking master.  This is a quick cleanup of some of the issues to get us in a better state rather than pushing them off.

**Comment by spbolton** [comment]
The issue was found to be related to https://github.com/dotCMS/core/pull/29242.   Made some cleanup which we could add but removing the priority of this cleanup as we are unblocked in the queue and should have a separate PR for fixing the code in the PR that triggered it

**Comment by nollymar** [comment]
Not required as we are moving to Semgrep
